puppet/manifests/apache.pp

```diff
@@ -184,7 +184,7 @@
       { 'path' => '^/pwa/(\w+)/(\d+)/rest/(keycloak).json$',
         'url'  => 'http://localhost:9050/rest/$3.json',
       },
-      { 'path' => '^/pwa/(\w+)/(\d+)/rest/(image-resources|init).json',
+      { 'path' => '^/pwa/(\w+)/(\d+)/rest/(image-resources.json|init.json|preferences)',
```

Comment:
That does not work since the internal URIs do not have the json suffix for init and image-resources. This way we would map `/pwa/javaland/2018/rest/init.json` to `/rest/init.json/javaland/2018`.